modules/reindex/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -20,6 +20,14 @@
 grant {
    // reindex opens socket connections using the rest client
    permission java.net.SocketPermission "*", "connect";
-   // rest client uses system properties which gets the default proxy
-   permission java.net.NetPermission "getProxySelector";
+};
+
+grant codeBase "${codebase.elasticsearch-rest-client-6.0.0-beta1-SNAPSHOT.jar}" {
+  // rest client uses system properties which gets the default proxy
+  permission java.net.NetPermission "getProxySelector";
+};
+
+grant codeBase "${codebase.httpasyncclient-4.1.2.jar}" {
+  // rest client uses system properties which gets the default proxy
+  permission java.net.NetPermission "getProxySelector";
```

Comment:
not required but I think it would be awesome if we could contribute a doPriviledged block to `httpasyncclient` such that down the road when we upgrade we can remove the block in our code